### PR TITLE
Fix documentation bugs and shortcut feature error messages

### PR DESCRIPTION
**Pull request by wongkokian**
You may view the modifications to the UG and DG here: https://wongkokian.github.io/tp/

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -621,15 +621,64 @@ You may use optional identifiers in conjunction with the minus(-) symbol to limi
 We understand that you might have commands that you frequently use. You may use shortcuts to abbreviate
 those commands to make using ClientBook more convenient.
 
+### <span style="color:#c9a500">Example Usage</span>
```
Comment (NBH99) [inline docs/UserGuide.md:620]: The example usage could be a few lines to describe the process with image 3.
Image 1 and 2 can go into the respective command description.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -894,10 +895,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 1a. The user input does not follow the format required.
+* 1a. The given arguments are invalid.
```
Comment (NBH99) [inline docs/DeveloperGuide.md]: The statement could be one or more of the given arguments are invalid.
The current statement could mean that only when all the given arguments are invalid then it gives error.

**Review by NBH99 — COMMENTED** [review]
Some minor issues, else LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (wongkokian) [reply docs/UserGuide.md:620]: Thanks! I have made the changes by shifting the images into the respective command descriptions.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (wongkokian) [reply docs/DeveloperGuide.md]: I have changed the statement to "At least one of the given arguments is invalid.".

**Pull request merged**